doc/sphinx/source/iio_osc/iio_osc.rst

```diff
@@ -35,10 +35,23 @@ Configure/Access Device Attributes (Parameters)
 ===============================================
 
 The IIO Oscilloscope allows user to access and configure different device parameters, 
-called as 'Attributes“. There are 2 types of attributes:
+called as 'Attributes“. There are 3 types of attributes:
 
 * Device Attributes (Global): Access/Configure common device parameters.
 * Channel Attributes (Specific to channels): Access/Configure channel specific device parameters.
+* Board Level Attributes (Can be Global and Channel specific): Configure the board level parameters. 
+  Projects only receive the board-level attribute when deemed necessary, thus it is not a standard inclusion
```

Comment:
nit: Will this be a good fit as a `Note`?

Reply:
I had the same thoughts, but there are too many note/warning sections back to back- I removed it after seeing the same. what do you suggest here?

Reply:
But I think it's better to have it as a note (which is clearly visible as a non-standard project specific inclusion) so that it takes out any confusion for APPS/users and they don't expect this to be added for all the projects!